### DISCUSSION TITLE
does not work on android devices

### DISCUSSION THREAD

**Discussion by effe-7** [tomasrudh/analogclock#34] [Q&A]
this card is very interesting but I have a problem.
the card works correctly both on IOS devices and via browser but on android device (specifically Sonoff NSPANEL pro) it doesn't work, you only see the yaml code that makes it up.
any solution?
thanks

**Comment by tomasrudh** [comment]
That's weird, I have no experience with the NSPANEL but I have a Samsung tablet that works fine, not sure why not for you. Could you share a screenshot of how it looks?

**Comment by effe-7** [comment]
Thank you for your quick feedback.
![IMG_6017](https://github.com/user-attachments/assets/6eac9179-079c-42b0-a861-096c01532730)

**Reply by tomasrudh** [reply]
Have you tried to clear the cache in the panel? I am not sure how to do that though.

**Reply by effe-7** [reply]
yes, I cleared the cache and also tried creating the tab several times.

**Reply by tomasrudh** [reply]
I am not sure what more to do. As it works for on other devices is the problem most probably on the NSPANEL device. See if you can find something in regards to that. For example, is javascript running properly? Can the cache be cleared in some other way?
